src/mcp_server_helm/commands/repository.py

```
@@ -0,0 +1,89 @@
+    """
+    logger.info(f"Running helm repo index with directory={directory}, url={url}")
+
+    cmd = ["helm", "repo", "index", directory]
+
+    if url:
+        cmd.extend(["--url", url])
+
+    if merge:
+        cmd.extend(["--merge", merge])
+
+    return execute_helm_command(cmd)
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)